src/content/reference/feature-flags.mdx

```diff
@@ -29,6 +29,7 @@ Below is a list of those variables that you can use to leverage these newer and
 | OKTETO_KUBERNETES_TIMEOUT                        | Specifies the timeout while deploying                                                                                                                                                                                           | `0`                    |
 | OKTETO_LOCAL_REGISTRY_STORE_PRIORITY_ENABLED     | Gives more priority to the local registry credentials than the Okteto Registry Credentials                                                                                                                                      | `false`                |
 | OKTETO_SMART_BUILDS_ENABLED                      | Enables the [Smart Builds feature](core/build-service.mdx#smart-builds) that uses a hash of the build context to determine if an image needs to be rebuilt or can be pulled from the registry. This can accelerate build times  | `true`                 |
+| OKTETO_SMART_BUILDS_IGNORE_FILES_ENABLED         | If set, considers ignores rules in .oktetoignore and .dockerignore for calculating smart build context.                                                                                                                         | `false`                |
```

Comment:
why do we consider `.oktetoignore`? smart builds only works with `dockerignore` files, no?

Reply:
is there a valid case for `.oktetoignore`? I think it's a bug if we are considering it, but maybe I am missing something